package.json

```diff
@@ -12,7 +12,7 @@
     "tmp-promise": "^1.0.4"
   },
   "scripts": {
-    "postinstall": "lerna clean --yes && lerna exec -- yarn --pure-lockfile --no-progress --mutex network",
+    "postinstall": "lerna exec -- yarn --pure-lockfile --no-progress --mutex network",
```

Comment:
20 seconds better. It s an old fix when we were using the lerna bootstrap.
Now we are all on yarn workspace so let remove the clean